Makefile

```diff
@@ -102,15 +91,18 @@ test-sec: gosec
 	@gosec -severity medium -confidence medium -exclude G304 -quiet ./...
 
 deploy: deploy-image
-	hack/deploy.sh
-.PHONY: deploy
+	LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY=127.0.0.1:5000/openshift/elasticsearch-operator-registry \
+	$(MAKE) elasticsearch-catalog-build && \
+	IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY=image-registry.openshift-image-registry.svc:5000/openshift/elasticsearch-operator-registry \
```

Comment:
if we're setting this here, doesn't that make it so for local we aren't pushing to the `LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY` ?

we set the port forwarding but then try to push to the remove registry...

Reply:
> if we're setting this here, doesn't that make it so for local we aren't pushing to the `LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY` ?
> 
> we set the port forwarding but then try to push to the remove registry...

The original catalog build script does a build and push using the same env var.  The issue is, however, the generally you don't have access to the registry and thus the reason we portforward to localhost.  This provides a mechanism by which the CI should be satisfied along with our local dev experience